micronaut-camunda-bpm-feature/src/test/kotlin/info/novatec/micronaut/camunda/bpm/feature/test/DefaultPropertySourceLoaderTest.kt

```diff
@@ -38,17 +39,19 @@ class DefaultPropertySourceLoaderTest {
         @Test
         fun `default value`() {
             assertEquals(50, applicationContext.environment.getProperty("datasources.default.maximum-pool-size", Int::class.java).get())
+            assertEquals(10, applicationContext.environment.getProperty("datasources.default.minimum-idle", Int::class.java).get())
```

Comment:
is there any benefit of adding the second property here? For "OverwrittenPropertyValue" we're only testing with one property so that should be sufficient here.